### PR TITLE
depricate yarn usage, v14 node, user friendly repo setup instructions

### LINKED ISSUE #47: Repo setup installation issue
Tried installing npm dependencies but does not work on my machine. Really want to get this working since this project is amazing.

Hit me up on Discord please 0xKudo#9889

### PR DESCRIPTION
**Pull request by mattstam**
## **Description**

`nodegit` comes with a bunch of potential NPM / Yarn installation issues (mostly stemming from `node-gyp` and `node-pre-gyp`). For this reason, we need to keep install instructions as minimal as possible to be compatible with many different setups (Mac, Linux, Windows).

## **Test Coverage**

We ran through the [Setup](https://github.com/git-consensus/contracts#setup) guide on Mac (Big Sur), Linux (Ubuntu 20.04), and Windows: install worked and `npm run test` passed for all.

- [X] My code follows the [Style Guide](../CONTRIBUTING.md#style-guide).
- [ ] My code requires a [Documentation Update](../CONTRIBUTING.md#generate-contract-api-docs).

Closes #47. Thanks to @0xKudo for pointing this out and being super helpful in testing this new method.

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -101,7 +95,7 @@ function getChainConfig(chain: keyof typeof chainIds): NetworkUserConfig {
     return {
         accounts: {
             count: ACCOUNT_COUNT,
-            mnemonic: MNEMONIC,
+            mnemonic: process.env.MNEMONIC,
```
Comment (0xKudo) [inline hardhat.config.ts:98]: better to keep ref on const

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -138,12 +143,12 @@ Getting fully prepared may involve getting a [INFURA_API_KEY](https://infura.io/
...
 
 ```sh
-yarn deploy --network goerli
+npm run deploy --network goerli
 ```
 
 #### [Generate contract API docs](#generate-contract-api-docs)
 
-Ensure `../docs` path exists for [the docs repo](https://github.com/git-consensus/docs). If it *isn't*:
+Ensure `../docs` path exists for [the docs repo](https://github.com/git-consensus/docs). If it _isn't_:
```
Comment (0xKudo) [inline CONTRIBUTING.md:151]: should be consistent here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -42,74 +44,77 @@ slither.config.json - "Configure Slither"
 tsconfig.json - "Configure Typescript"
 ```
 
---- *(not an extensive list of all files)* ---
+--- _(not an extensive list of all files)_ ---
```
Comment (0xKudo) [inline CONTRIBUTING.md:47]: same

**Review by 0xKudo — CHANGES_REQUESTED** [review]

File: hardhat.config.ts (hunk 1 above)
Comment (mattstam) [reply hardhat.config.ts:91]: They're all the same now.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mattstam) [reply CONTRIBUTING.md]: yep fixed

**Review by 0xKudo — APPROVED** [review]

**Pull request merged**